reconcile/utils/mr/glitchtip_access_reporter.py

```diff
@@ -0,0 +1,122 @@
+from collections.abc import Sequence
+from datetime import date
+from datetime import datetime as dt
+from datetime import timezone
+from pathlib import Path
+
+from jinja2 import Template
+from pydantic import BaseModel
+
+from reconcile.utils.gitlab_api import GitLabApi
+from reconcile.utils.mr.base import MergeRequestBase
+from reconcile.utils.mr.labels import AUTO_MERGE
+
+CURRENT_USERS_TABLE_TEMPLATE = """
+| Username | Name | Organizations and Access Levels |
+| -------- | ---- | ------------------------------- |
+{% for user in users|sort -%}
+| {{ user.username }} | {{ user.name }} |{% for org in user.organizations %} {{org.name}} ({{ org.access_level }}){% if not loop.last%},{% endif %}{% endfor %} |
+{% endfor %}
+""".strip()
+
+
+class GlitchtipAccessReportOrg(BaseModel):
+    name: str
+    access_level: str
+
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, str):
+            raise NotImplementedError("Cannot compare to non string objects.")
+        return self.name == other
```

Comment:
can you elaborate why an `GlitchtipAccessReportOrg` is compared to a string?

Reply:
I want to [compare to a string](https://github.com/app-sre/qontract-reconcile/pull/3372/files#diff-2d4f31b8cea30e31276115b5201a33f5f37ecf57187e2799ee4bb6d7afe7a90bR62); therefore I implemented it this way.

Reply:
`__eq__` is not acting symmetricly, so `GlitchtipAccessReportOrg() == "some string"` yields not the same results as `"some string" == GlitchtipAccessReportOrg()`. 

to make reading the code easier for us, can you change it to something like this?

```
org.name not in {o.name for o in report_user.organizations}
```